samples/net/sockets/echo_client/overlay-ot.conf

```diff
@@ -9,7 +8,6 @@ CONFIG_NET_IPV4=n
 
 CONFIG_NET_IPV6_NBR_CACHE=n
 CONFIG_NET_IPV6_MLD=n
-CONFIG_NET_CONFIG_NEED_IPV6=y
 CONFIG_NET_CONFIG_NEED_IPV4=n
 CONFIG_NET_CONFIG_MY_IPV4_ADDR=""
 CONFIG_NET_CONFIG_PEER_IPV4_ADDR=""
```

Comment:
We should also investigate CONFIG_NET_TX_STACK_SIZE and CONFIG_NET_RX_STACK_SIZE, since the major RX/TX processing was moved to the OpenThread thread, they probably don't need to be that large anymore.

Reply:
samples seem to work on defaults now